CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 12.2.0 "corduroy-aardvark" (2021-08-04)
+
+### cdk
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **overlay:** expand `OverlayReference` interface to cover more properties ([#23254](https://github.com/angular/components/issues/23254)) ([efa9e92a789e6647f7428c58ad720104007becfc](https://github.com/angular/components/commit/efa9e92a789e6647f7428c58ad720104007becfc)), closes [#23234](https://github.com/angular/components/issues/23234) |
+| bug fix |  **text-field:** avoid page jump on firefox ([#23296](https://github.com/angular/components/issues/23296)) ([ef18df02d20f69c092503ca6a164afd3e81dfbc8](https://github.com/angular/components/commit/ef18df02d20f69c092503ca6a164afd3e81dfbc8)), closes [#23233](https://github.com/angular/components/issues/23233) |
+| performance |  **drag-drop:** avoid excessive change detections with zone-patch-rxjs ([#23272](https://github.com/angular/components/issues/23272)) ([7d29bc301e7dd21fe601ea1c291872abfa3fc939](https://github.com/angular/components/commit/7d29bc301e7dd21fe601ea1c291872abfa3fc939)), closes [#23248](https://github.com/angular/components/issues/23248) |
+
+### material
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **core:** handle hues inferred as numbers ([#23237](https://github.com/angular/components/issues/23237)) ([a752641ffff853864f87ff81c5fc40192120b62a](https://github.com/angular/components/commit/a752641ffff853864f87ff81c5fc40192120b62a)), closes [#23230](https://github.com/angular/components/issues/23230) |
+
+### material-experimental
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **mdc-paginator:** add screen reader announcement for current range when navigating between pages ([#23285](https://github.com/angular/components/issues/23285)) ([8d4f4e245d62ae85ab9e71cc9c8010db318a98e8](https://github.com/angular/components/commit/8d4f4e245d62ae85ab9e71cc9c8010db318a98e8)) |
+| bug fix |  **mdc-table:** flex tables should stretch cells ([#23259](https://github.com/angular/components/issues/23259)) ([47a9c532d73b29d0d6488c0e9187eb0a8d27580a](https://github.com/angular/components/commit/47a9c532d73b29d0d6488c0e9187eb0a8d27580a)) |
+
+
 # 12.2.0-rc.0 "schmaltz-waltz" (2021-07-28)
```

Comment:
Please merge the points for this release into the new one and delete the rc. (This is what we typically do for final releases after some next/rc)